backend/services/deviceconnect/api/http/device.go

```diff
@@ -240,15 +240,7 @@ func (h DeviceController) connectWSWriter(
 		return err
 	}
 
-	pingPeriod := (pongWait * 9) / 10
-	ticker := time.NewTicker(pingPeriod)
-	defer ticker.Stop()
-	conn.SetPongHandler(func(string) error {
-		ticker.Reset(pingPeriod)
-		return conn.SetReadDeadline(time.Now().Add(pongWait))
-	})
 	conn.SetPingHandler(func(msg string) error {
-		ticker.Reset(pingPeriod)
 		err := conn.SetReadDeadline(time.Now().Add(pongWait))
```

Comment:
Since the deadline is never set anywhere else, this would lead to the breaking the connection after a minute unless the client sends another ping.
```suggestion
```

Reply:
There's another one on line 237:
https://github.com/mendersoftware/mender-server/blob/ceb2a79f4b41bc014f5f91094f44f682b1cbc986/backend/services/deviceconnect/api/http/device.go#L237-L241